docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '2'
+
+services:
+  ntt:
+    depends_on:
+        - postgres
+    build:
+      context: .
+    restart: on-failure
+    environment:
+      - NEW_BUILD=true
+      - POSTGRES_HOST=postgres
+    ports:
+      - "5001:5001"
+
+  postgres:
+    image: postgres:14.2
+    healthcheck:
+      test: [ "CMD", "pg_isready", "-q", "-d", "postgres", "-U", "root" ]
+      timeout: 45s
+      interval: 10s
+      retries: 10
+    restart: always
+    environment:
+      - POSTGRES_USER=root
+      - POSTGRES_PASSWORD=password
+      - APP_DB_USER=ubuntu
+      - APP_DB_PASS=ubuntu
+      - APP_DB_NAME=newtrains
+    volumes:
+      - ./db:/docker-entrypoint-initdb.d/
+    expose:
+      - "5432"
+
+networks:
+  default:
+    name: transit
```

Comment:
Nit: Docker compose creates a network by default, no?

Reply:
yeah but I wanted to name it specifically.